editor/editor_version_control_settings.h

```diff
@@ -0,0 +1,46 @@
+#ifndef EDITOR_VERSION_CONTROL_SETTINGS_H
+#define EDITOR_VERSION_CONTROL_SETTINGS_H
+
+#include "core/array.h"
+#include "editor/editor_data.h"
+#include "editor/editor_vcs_api.h"
+#include "scene/gui/dialogs.h"
+
+class EditorVersionControlSettings : public VBoxContainer {
```

Comment:
Maybe all this call should be all moved to the editor plugin. To be discussed ;)

Reply:
Shifting this into editor plugin is requiring me to include a plugin file inside `editor_node.h`, since there are no plugins being currently imported, would it be okay to add one? Or maybe I forward-declare this near the top of `editor_node.h`. 

Either way, `editor_node.h` ends up with having a dependency on a plugin. That seems to be a little bit off for the design of a class that manages all plugins and requires no knowledge of the plugin implementation.